pkg/commands/requests/actions/new.go

```diff
@@ -65,16 +71,20 @@ func Create() *cobra.Command {
 				return err
 			}
 
-			newAccessRequest, err := waitForNewRequest(cmd, client, &creationTime)
+			var newAccessRequest *clientapi.AccessRequestClientModel
+
+			newAccessRequest, err = requestloader.RunRequestLoader(cmd.Context(), client, creationTime, cmdFlags.timeout, cmdFlags.noWait)
 			if err != nil {
 				return err
 			}
 
 			table := services.GenerateRequestsTable([]clientapi.AccessRequestClientModel{*newAccessRequest})
 
-			_, err = fmt.Fprintln(cmd.OutOrStdout(), "")
-			if err != nil {
-				return err
+			if !cmdFlags.dontRunInteractive {
+				_, err = fmt.Fprintln(cmd.OutOrStdout(), "")
+				if err != nil {
+					return err
+				}
```

Comment:
when using the interactive mode, the output look weird when glued together (the request table and the selected option from the form) so I added newline, but when I think about it, I will print it to regular stdout instead of the command stdout.